WordPress/src/main/java/org/wordpress/android/ui/activitylog/list/filter/ActivityLogTypeFilterFragment.kt

```diff
@@ -31,6 +39,35 @@ class ActivityLogTypeFilterFragment : DialogFragment() {
     }
 
     override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View? {
-        return inflater.inflate(layout.activity_log_type_filter_fragment, container, false)
+        return inflater.inflate(R.layout.activity_log_type_filter_fragment, container, false)
+    }
+
+    override fun getTheme(): Int {
+        return R.style.WordPress_FullscreenDialog
+    }
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+        initToolbar(view)
+        initRecyclerView()
+    }
+
+    private fun initToolbar(view: View) {
+        val toolbar = view.findViewById(R.id.toolbar_main) as MaterialToolbar
```

Comment:
Question: For `recycler_view` the Kotlin Android Extensions synthetic is used. However, in this case `findViewById(...)` is used instead. Why is that? I would suggest one or the other for consistency purposes. Wdyt?

PS: I am inclining more on using synthetics for now (although deprecated) in order to make any future migration a bit easier. Also, this will help us be consistent with the rest of the codebase. I am guessing synthetics are now being utilised more than `findViewById(...)`, right? If not, then the opposite might be better. Not sure. Wdyt?

Reply:
Copy pasting mistake :) - fixed in 052cb4497a9cdc6a8883808f084520e98848a69d